doc/flatpak-manifest.xml

```diff
@@ -531,10 +531,22 @@
                         <term><option>url</option> (string)</term>
                         <listitem><para>The URL of a remote archive that will be downloaded. This overrides path if both are specified.</para></listitem>
                     </varlistentry>
+                    <varlistentry>
+                        <term><option>md5</option> (string)</term>
+                        <listitem><para>The md5 checksum of the file, verified after download</para></listitem>
+                    </varlistentry>
```

Comment:
It would be good to mention somewhere that SHA-256 is still the preferred/default checksum type, and that MD5 and SHA-1 are discouraged and shouldn’t be relied on if possible.